koch.nim

```diff
@@ -428,6 +430,48 @@ proc xtemp(cmd: string) =
   finally:
     copyExe(d / "bin" / "nim_backup".exe, d / "bin" / "nim".exe)
 
+proc runCI(cmd: string) =
+  doAssert cmd.len == 0, cmd # avoid silently ignoring
+  echo "runCI:", cmd
+  # todo: consider replacing ./koch commands below with direct proc call, eg:
```

Comment:
Remove this "todo" and turn it into a comment saying "Do not replace these commands with direct proc calls as that would weaken our testing efforts".

Reply:
fixed